### PR TITLE
Add @xtuchyna as maintainer

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Related to #1295 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

`mi` and `mi-scheduler` related code is present both in `workflows.py` and also `openshift.py` files, therefore I suggest adding myself as a maintainer, so I can be able to make release requests, too

### CODE REVIEW

**Comment by xtuchyna** [comment]
Could you also lgtm this @goern ? thx!

File: OWNERS [hunk 1]
```diff
@@ -5,6 +5,7 @@ approvers:
   - harshad16
   - mayaCostantini
   - sesheta
+  - xtuchyna
```
Comment (harshad16) [inline OWNERS:8]:
```suggestion
  - xtuchyna
  - goern 
```

somehow we missed including Christoph, can you please add him too only on the approver's list.

Comment (xtuchyna) [reply OWNERS:8]: All good now @harshad16 :+1:

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

Thanks :100: 
Welcome as approver and reviewer.

**Pull request merged**